src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User nullUser = null;
+    private static final User nullLoginUser = new User(null, "1234567", 11);
+    private static final User nullPasswordUser = new User("????", null, 16);
+    private static final User nullAgeUser = new User("????", "????????", null);
+    private static final User illegalAgeUser = new User("Pudj2008", "asdfg2002", 13);
```

Comment:
or testing it is better to take marginally unacceptable input data, in this case, 17 years